include/ps/internal/van.h

```diff
@@ -93,7 +93,7 @@ class Van {
    * different nodes on the same machine picked the same port
    * \return return the port binded, -1 if failed.
    */
-  virtual int Bind(const Node &node, int max_retry) = 0;
+  virtual int Bind(Node& node, int max_retry) = 0;
```

Comment:
should it return void? Otherwise what value has to be returned?

Reply:
Just updated the docs. Currently it just returns ports[0] for backward compatibility of exisitng vans